examples/force_handle_mixin.py

```diff
@@ -0,0 +1,94 @@
+"""
+Create Pipeline Steps that only require implementing handler methods.
+========================================================================================================================
+
+If a pipeline step only needs to implement handler methods, then you can inherit from the ForceHandleMixin as demonstrated here.
+Handler methods are useful when :
+- You need to change the shape of the data container passed to the following steps, or the wrapped steps.
+- You want to apply side effects based on the data container, and the execution context.
+- You want to change the pipeline execution flow based on the data container, and the execution context.
```

Comment:
You must add a single space before the `- ` to make the list work, because with our docstring type (RST), lists needs a space before the dash.